packages/nocodb/src/lib/db/sql-data-mapper/lib/sql/BaseModelSqlv2.ts

```diff
@@ -1861,7 +1875,7 @@ class BaseModelSqlv2 {
     await this.handleHooks('After.insert', data, req);
     // if (req?.headers?.['xc-gui']) {
     const id = this._extractPksValues(data);
-    Audit.insert({
+    await Audit.insert({
```

Comment:
you can remove the await here since we don't want to wait for the audit insert

Reply:
@pranavxc It will fail some test, since we are resetting sakila in test env based on this, at least in the case of test we need the await.

Reply:
See if you can manage it with a delay

Reply:
@pranavxc We can to await only if env is test maybe? But will clutter the code base

Reply:
If it's using for data verification then you can directly check it in the database